includes/Framework/Api/Base.php

```diff
@@ -620,12 +621,12 @@ protected function set_request_headers( array $headers ) {
 	/**
 	 * Set HTTP basic auth for the request
 	 *
-	 * Since 2.2.0
+	 * @since 2.2.0
 	 * @param string $username
 	 * @param string $password
 	 */
 	protected function set_http_basic_auth( $username, $password ) {
-		$this->request_headers['Authorization'] = sprintf( 'Basic %s', base64_encode( "{$username}:{$password}" ) );
+		$this->request_headers['Authorization'] = sprintf( 'Basic %s', base64_encode( "{$username}:{$password}" ) ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.obfuscation_base64_encode
```

Comment:
FYI it is also possible to add phpcs:ignore above the line which is ignored because sometimes adding it to the end makes the string too long.